app/main.py

```diff
@@ -1,6 +1,22 @@
+import os
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    base_url = "http://api.weatherapi.com/v1"
+    api_key = os.environ["API_KEY"]
+    city = "Paris"
+
+    url = base_url + "/current.json?key=" + api_key + "&q=" + city
```

Comment:
Does that make sense to put `/current.json ` to `base_url` as a query parameters in url are start exactly with `?`